classes/order/Order.php

```diff
@@ -1088,8 +1088,6 @@ public static function getCustomerNbOrders($id_customer)
      */
     public static function getOrderByCartId($id_cart)
     {
-        Tools::displayAsDeprecated('Use Order::getIdByCartId() instead');
```

Comment:
For some reason having this line made the travis build fail. after I removed it things started building again... not sure why?